code/extensions/CommentsExtension.php

```diff
@@ -56,6 +61,8 @@ class CommentsExtension extends DataExtension {
 		'html_allowed' => false,
 		'html_allowed_elements' => array('a', 'img', 'i', 'b'),
 		'use_preview' => false,
+		'nested_comments' => false,
```

Comment:
By default this is disabled to prevent upgrades to this module "automatically" adding nesting. This might be enabled by default in the future.